src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -230,30 +241,47 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             }
         }
 
-        if (isSetAsideApplicationRequest(sscsCaseData)) {
+        if (isSetAsideApplication(sscsCaseData)) {
             sscsCaseData.setState(State.POST_HEARING);
             sscsCaseData.setInterlocReviewState(REVIEW_BY_JUDGE);
             if (PartyItemList.DWP.getCode().equals(sscsCaseData.getOriginalSender().getValue().getCode())) {
                 sscsCaseData.setDwpState(DwpState.SET_ASIDE_REQUESTED);
             }
         }
 
+        if (isCorrectionApplication(sscsCaseData)) {
+            sscsCaseData.setState(State.POST_HEARING);
+            sscsCaseData.setDwpState(DwpState.CORRECTION_REQUESTED);
```

Comment:
add new line before if
and get the conditions of IFs statement to separate vars for readability

Reply:
done